docs/gui/liquids.rst

```diff
@@ -5,13 +5,14 @@ gui/liquids
     :summary: Interactively paint liquids onto the map.
     :tags: fort armok map
 
-This tool is a gui for spawning water or magma. Allowing you to click and paint
-liquid onto the map.
+This tool is a gui for spawning water, magma, and river sources on the map.
+Allowing you to click and paint liquid onto the map. It can optionally clean
```

Comment:
This appears to be a sentence fragment. Perhaps combine with the previous sentence?